docs/concepts/admission-webhook.md

```diff
@@ -0,0 +1,45 @@
+# Lokomotive admission webhooks
+
+As part of the cluster control plane Lokomotive creates additional admission webhooks, which add extra features to the cluster. This document describes the webhooks we install and what they do.
+
+## Default service account mutating webhook
```

Comment:
I find this section hard to follow due to the ordering of information in it. I would use the following order:

- Pods are assigned an SA by default.
- This SA grants access to the API server, which is risky.
- Your application likely doesn't need access to the API server so we can eliminate the risk.
- The webhook eliminates the risk by disabling automounting of the SA.
- Here is how you check which SA is attached to a pod.
- For more info visit the k8s docs.